src/OpenApi/Serializer/OpenApiNormalizer.php

```diff
@@ -24,8 +24,10 @@
 final class OpenApiNormalizer implements NormalizerInterface, CacheableSupportsMethodInterface
 {
     public const FORMAT = 'json';
-    public const OPEN_API_PRESERVE_EMPTY_OBJECTS = 'open_api_preserve_empty_objects';
+    public const UNSET_VALUE = 'unset this value';
```

Comment:
I'm not sure to understand what this does.

Reply:
By default the `security`node on an Operation should not be added if it's not set. Still, if it's set as an empty value it should stay: 

> A declaration of which security mechanisms can be used for this operation. The list of values includes alternative security requirement objects that can be used. Only one of the security requirement objects need to be satisfied to authorize a request. This definition overrides any declared top-level security. To remove a top-level security declaration, an empty array can be used.

Reply:
Couldn't we use the `null` value to distinguish from the empty array instead (and remove the they if it is null)?